articles/azure-app-configuration/enable-dynamic-configuration-aspnet-core.md

```diff
@@ -348,4 +351,4 @@ A *sentinel key* is a special key used to signal when configuration has changed.
 In this tutorial, you enabled your ASP.NET Core web app to dynamically refresh configuration settings from App Configuration. To learn how to use an Azure-managed identity to streamline the access to App Configuration, continue to the next tutorial.
 
 > [!div class="nextstepaction"]
-> [Managed identity integration](./howto-integrate-azure-managed-service-identity.md)
\ No newline at end of file
+> [Managed identity integration](./howto-integrate-azure-managed-service-identity.md)
```

Comment:
I don't see any changes in this line but it shows as diff.

Reply:
Most of our docs miss an empty line at the end, so a diff is shown when any part of the doc is updated.